docs/source/commandline/commandline-all.rst

```diff
@@ -928,95 +1148,176 @@ Core Video Options
+
+    This option is intended mainly for those who have tweaked their video card's
+    settings to provide carefully matched refresh rate options.  Note that this
+    option does not work with **-video gdi** mode.
+
+    The default is OFF (**-nosyncrefresh**).
 
 .. _mame-commandline-prescale:
 
 **-prescale** *<amount>*
 
-	Controls the size of the screen images when they are passed off to the graphics system for scaling. At the minimum setting of 1, the screen is rendered at its original resolution before being scaled. At higher settings, the screen is expanded by a factor of *<amount>* before being scaled. With **-video d3d**, this produces a less blurry image at the expense of some speed. The default is ``1``.
+    Controls the size of the screen images when they are passed off to the
+    graphics system for scaling.  At the minimum setting of 1, the screen is
+    rendered at its original resolution before being scaled.  At higher
+    settings, the screen is expanded by a factor of *<amount>* before being
+    scaled.  With **-video d3d**, this produces a less blurry image at the
+    expense of some speed.
 
-	This is supported with all video output types (``bgfx``, ``d3d``, etc) on Windows and is **ONLY** supported with OpenGL on other platforms.
+    The default is ``1``.
+
+    This is supported with all video output types (``bgfx``, ``d3d``, etc) on
+    Windows and is **ONLY** supported with OpenGL on other platforms.
 
 .. _mame-commandline-filter:
 
 **-[no]filter** / **-[no]d3dfilter** / **-[no]flt**
 
-	Enable bilinear filtering on the system screen graphics. When disabled, point filtering is applied, which is crisper but leads to scaling artifacts. If you don't like the filtered look, you are probably better off increasing the **-prescale** value rather than turning off filtering altogether. The default is ON (**-filter**).
+    Enable bilinear filtering on the system screen graphics.  When disabled,
+    point filtering is applied, which is crisper but leads to scaling artifacts.
+    If you don't like the filtered look, you are probably better off increasing
+    the **-prescale** value rather than turning off filtering altogether.
+
+    The default is ON (**-filter**).
 
-	This is supported with OpenGL and D3D video on Windows and is **ONLY** supported with OpenGL on other platforms.
+    This is supported with OpenGL and D3D video on Windows and is **ONLY**
+    supported with OpenGL on other platforms.
 
-	Use bgfx_screen_chains with BGFX to adjust filtering with that video system.
+    Use ``bgfx_screen_chains`` in your INI file(s) to adjust filtering with the
+    BGFX video system.
 
 .. _mame-commandline-unevenstretch:
 
 **-[no]unevenstretch**
 
-	Allow non-integer stretch factors allowing for great window sizing flexability. The default is ON. (**-unevenstretch**)
+    Allow non-integer stretch factors allowing for great window sizing
+    flexability.
+
+    The default is ON. (**-unevenstretch**)
```

Comment:
For the scaling options I've tried to clean up the use of "stretch" vs "scale" in the `-showusage` output - the same should probably be done here (this option and the ones above it).  See 113f98accc754abe34ce8825ca179a0afb197281

Reply:
Noted and fixed.